src/components/atoms/AtomInput.vue

```diff
@@ -115,4 +115,9 @@ export default {
 };
 </script>
 
-<style></style>
+<style lang="scss">
+.label {
+    font-size: 14px !important;
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#Important_Declarations

Reply:
needed to override any existed css